demo/Views/SeekBarView.vala

```diff
@@ -19,11 +19,96 @@
  */
 
 public class SeekBarView : Gtk.Grid {
+    private Gtk.Popover preview_popover;
+    private Gtk.Label preview_label;
+
+    public SeekBarView () {
+        Object (valign: Gtk.Align.CENTER,
+                margin: 24);
```

Comment:
Is there a reason why these are not set in the `construct` method?

Reply:
`Object ()` is here to set Object properties ;)

Reply:
Sure, but it's hardcoding those values. I thought we're keeping all UI related code in the `construct`?